### PR TITLE
Ctor 398 backbox

### PR DESCRIPTION
**Pull request by sdepassio**
## Description

New monitoring connector Backbox

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: pp/sidebarsPp.js [hunk 1]
```diff
@@ -1484,6 +1484,10 @@ module.exports = {
           type: 'doc',
           id: 'integrations/plugin-packs/procedures/network-atto-fibrebridge-snmp'
         },
+        {
+          type: 'doc',
+          id: 'integrations/plugin-packs/procedures/network-backbox-rest'
```
Comment (cg-tw) [inline pp/sidebarsPp.js]:
```suggestion
          id: 'integrations/plugin-packs/procedures/network-backbox-restapi'
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
...
+id: network-backbox-restapi
+title: Backbox
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Dépendances du Connecteur de supervision
+
+Les connecteurs de supervision suivants sont automatiquement installés lors de l'installation du connecteur **Backbox** 
+depuis l'interface web et le menu **Configuration > Gestionnaire de connecteurs de supervision** :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:11]:
```suggestion
depuis la page **Configuration > Gestionnaire de connecteurs de supervision** :
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
+---
+id: network-backbox-restapi
+title: Backbox
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:8]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+| *intellichecks*#intellicheck.success.count | count |
+| *intellichecks*#intellicheck.suspect.count | count |
+| *intellichecks*#intellicheck.failure.count | count |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:77]: prérequis

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 5]
```diff
@@ -0,0 +1,413 @@
+---
+id: network-backbox-restapi
+title: Backbox
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Connector Dependencies
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:8]:
```suggestion
## Connector dependencies
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+| *intellichecks*#intellicheck.success.count | count |
+| *intellichecks*#intellicheck.suspect.count | count |
+| *intellichecks*#intellicheck.failure.count | count |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+*Specify prerequisites that are relevant. You may want to just provide a link\n\
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:76]: prerequisites

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+| CRITICALFAILURE | Set critical threshold for 'failure'                                                               |                   |             |
+| WARNINGSUCCESS  |                                                                                                    |                   |             |
+| CRITICALSUCCESS |                                                                                                    |                   |             |
+| WARNINGSUSPECT  | Set warning threshold for 'suspect'                                                                |                   |             |
+| CRITICALSUSPECT | Set critical threshold for 'suspect'                                                               |                   |             |
+| WARNINGTOTAL    |                                                                                                    |                   |             |
+| CRITICALTOTAL   |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:203]:
```suggestion
| WARNINGTOTAL    | Set warning threshold for 'total'                                                                                                     |                   |             |
| CRITICALTOTAL   | Set warning threshold for 'total'                                                                                                     |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+|:----------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERTYPE      | Filter backups by type                                                                             |                   | X           |
+| REPORTID        | Specify report id                                                                                  |                   |             |
+| WARNINGFAILURE  | Set warning threshold for 'failure'                                                                |                   |             |
+| CRITICALFAILURE | Set critical threshold for 'failure'                                                               |                   |             |
+| WARNINGSUCCESS  |                                                                                                    |                   |             |
+| CRITICALSUCCESS |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:230]:
```suggestion
| WARNINGSUCCESS  | Set warning threshold for 'succes'                                                                                                   |                   |             |
| CRITICALSUCCESS | Set critical threshold for 'succes'                                                                                                     |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 9]
```diff
@@ -0,0 +1,414 @@
...
+| WARNINGSUSPECT  | Set warning threshold for 'suspect'                                                                |                   |             |
+| CRITICALSUSPECT | Set critical threshold for 'suspect'                                                               |                   |             |
+| WARNINGTOTAL    |                                                                                                    |                   |             |
+| CRITICALTOTAL   |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:234]:
```suggestion
| WARNINGTOTAL    | Set warning threshold for 'total'                                                                                                     |                   |             |
| CRITICALTOTAL   | Set warning threshold for 'total'                                                                                                     |                   |             |
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 10]
```diff
@@ -0,0 +1,413 @@
...
+| CRITICALFAILURE | Set critical threshold for 'failure'                                                               |                   |             |
+| WARNINGSUCCESS  |                                                                                                    |                   |             |
+| CRITICALSUCCESS |                                                                                                    |                   |             |
+| WARNINGSUSPECT  | Set warning threshold for 'suspect'                                                                |                   |             |
+| CRITICALSUSPECT | Set critical threshold for 'suspect'                                                               |                   |             |
+| WARNINGTOTAL    |                                                                                                    |                   |             |
+| CRITICALTOTAL   |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:204]:
```suggestion
| WARNINGTOTAL    | Set warning threshold for 'total'                                                                                                     |                   |             |
| CRITICALTOTAL   | Set warning threshold for 'total'                                                                                                     |                   |             |
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 11]
```diff
@@ -0,0 +1,413 @@
...
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:235]:
```suggestion
| WARNINGTOTAL    | Set warning threshold for 'total'                                                                                                     |                   |             |
| CRITICALTOTAL   | Set warning threshold for 'total'                                                                                                     |                   |             |
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 12]
```diff
@@ -0,0 +1,413 @@
...
+<Tabs groupId="sync">
+<TabItem value="Backup" label="Backup">
+
+| Macro           | Description                                                                                        | Default value     | Mandatory   |
+|:----------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERTYPE      | Filter backups by type                                                                             |                   | X           |
+| WARNINGFAILURE  | Set warning threshold for 'failure'                                                                |                   |             |
+| CRITICALFAILURE | Set critical threshold for 'failure'                                                               |                   |             |
+| WARNINGSUCCESS  |                                                                                                    |                   |             |
+| CRITICALSUCCESS |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:200]:
```suggestion
| WARNINGSUCCESS  | Set warning threshold for 'succes'                                                                                                   |                   |             |
| CRITICALSUCCESS | Set critical threshold for 'succes'                                                                                                     |                   |             |
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 13]
```diff
@@ -0,0 +1,413 @@
...
+|:----------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERTYPE      | Filter backups by type                                                                             |                   | X           |
+| REPORTID        | Specify report id                                                                                  |                   |             |
+| WARNINGFAILURE  | Set warning threshold for 'failure'                                                                |                   |             |
+| CRITICALFAILURE | Set critical threshold for 'failure'                                                               |                   |             |
+| WARNINGSUCCESS  |                                                                                                    |                   |             |
+| CRITICALSUCCESS |                                                                                                    |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:231]:
```suggestion
| WARNINGSUCCESS  | Set warning threshold for 'succes'                                                                                                   |                   |             |
| CRITICALSUCCESS | Set critical threshold for 'succes'                                                                                                     |                   |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 14]
```diff
@@ -0,0 +1,413 @@
...
+| *intellichecks*#intellicheck.success.count | count |
+| *intellichecks*#intellicheck.suspect.count | count |
+| *intellichecks*#intellicheck.failure.count | count |
+
+</TabItem>
+</Tabs>
+
+## Prérequis
+
+Pour utiliser le connecteur **Backbox**, vous devez generer un jeton API : rendez-vous sur votre tableau de bord Backbox puis sur la page Administrative / Authentication / API Tokens.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:77]:
```suggestion
Pour utiliser le connecteur **Backbox**, vous devez générer un jeton d'API : rendez-vous sur votre tableau de bord Backbox puis sur la page **Administrative > Authentication > API Tokens**.
```

File: pp/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 15]
```diff
@@ -0,0 +1,412 @@
...
+| *intellichecks*#intellicheck.success.count | count |
+| *intellichecks*#intellicheck.suspect.count | count |
+| *intellichecks*#intellicheck.failure.count | count |
+
+</TabItem>
+</Tabs>
+
+## Prerequisites
+
+To monitor a **Backbox** resource, you need to generate an API token: go to your Backbox dashboard and then on Administrative / Authentication / API Tokens page.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-backbox-restapi.md:76]:
```suggestion
To monitor a **Backbox** resource, you need to generate an API token: go to your Backbox dashboard and then to the **Administrative > Authentication > API Tokens** page.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md [hunk 16]
```diff
@@ -0,0 +1,413 @@
...
+Le jeton API est associé à un compte utilisateur. Il est recommandé de créer un compte utilisateur dédié à la supervision.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-backbox-restapi.md:78]:
```suggestion
Le jeton d'API est associé à un compte utilisateur. Il est recommandé de créer un compte utilisateur dédié à la supervision.
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by lucie-dubrunfaut — APPROVED** [review]

**Pull request merged**